### PR TITLE
Fixing Recognition Problems

### PR DESCRIPTION
**Pull request by calvinc64**

### CODE REVIEW

**Comment by calvinc64** [comment]
Sorry, there are some Errors, tried to fix them with code from the other apps, but yeah.... you will see what Ive tried.

**Comment by calvinc64** [comment]
So, Just looked at the Errors from Tarvis, now fixed ^^

File: src/connectors/listen.moe.js [hunk 1]
```diff
@@ -3,7 +3,14 @@
 Connector.playerSelector = '#app';
 
 Connector.artistSelector = '.playerContainer.columns.mini .player .player-song-artist';
-
 Connector.trackSelector = '.playerContainer.columns.mini .player .player-song-title';
 
+Connector.getArtistTrack = () => {
+	return $('.player-song-title')[0].innerText.split(' [')[0];
```
Comment (inverse) [inline src/connectors/listen.moe.js]: Can you give some examples of tracks which this helps?

File: src/connectors/listen.moe.js [hunk 2]
```diff
@@ -3,7 +3,14 @@
...
+};
+
+Connector.getArtist = () => {
+	return $('.player-song-artist-container')[0].innerText.replace(' ,', ',');
```
Comment (inverse) [inline src/connectors/listen.moe.js]: Same with this one.

File: src/connectors/listen.moe.js (hunk 1 above)
Comment (calvinc64) [reply src/connectors/listen.moe.js]: Ok... The page sometimes displays the Show (From the Series the Opening is from) and this is in this format: artist - Trackname [Some aditional show info that confuses lastfm]
and in the artist field if multiple groups did work on one track is weird too... let me search an example...

Here an nice example, where both the title and the Artists are bad:

**Comment by inverse** [comment]
@calvinc64 any update?

**Comment by calvinc64** [comment]
Sorry for not leaving any update for so long, did not have time enough to spend for git at the moment, but I will look into the Problem this weekend again!

**Comment by calvinc64** [comment]
Well now its just showing me this...
grafik
And yeah did test it on Firefox 66.0.3 (64B on Windows 10)

**Comment by alexesprit** [comment]
@calvinc64 It was fixed in v2.16.3.

**Comment by alexesprit** [comment]
I'm closing this one as outdated.

**Pull request closed without merging**